src/ExperimentTable.js

```diff
@@ -59,6 +65,15 @@ class ExperimentTable extends React.Component {
           .includes(searchQuery.toLowerCase())
       )
   }
+  //this function checks if value exists in json object e.g.
+  //jsonObject = {a: 1, b: 2} then containsValue(jsonObject, 1) will return true
+  //jsonObject = {a: 1, b: 2} then containsValue(jsonObject, 3) will return false
+  containsValue(jsonObject, value) {
+    return Object.keys(jsonObject).some(key =>
+      typeof jsonObject[key] === `object` ?
+        this.containsValue(jsonObject[key], value) : jsonObject[key] === value
+    )
+  }
```

Comment:
The key point of this function is that it looks for a value *deeply*. Otherwise you could just do `Object.values(jsonObject).includes(value)`.